### DISCUSSION TITLE
柱状图值为0不显示

### DISCUSSION THREAD

**Discussion by gungun0613** [antvis/G2#6029] [Q&A] [answered]
### 问题描述

柱状图值为0时不显示，也不显示tooltip

<img width="313" alt="image" src="https://github.com/antvis/G2/assets/32888772/2efd92ff-7670-424b-b4b6-14e16da4381a">

data数据为
[
  {
    year: '2023-01-01',
    sales: 110,
  },
  {
    year: '2023-02-01',
    sales: 450,
  },
  {
    year: '2023-03-01',
    sales: 560,
  },
  {
    year: '2023-04-01',
    sales: 0,
  },
  {
    year: '2023-05-01',
    sales: 0,
  },
  {
    year: '2023-06-01',
    sales: 0,
  },
];

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by hustcc** [comment, marked as answer]
```ts
.style('minHeight', 5)
```

为 0 的时候，数据映射成图形高度就是 0，符合预期的，如果实在需要显示出来，按照上面的配置。

**Comment by hustcc** [comment]
G2Plot 可以不显示图形，但是显示 tooltip，有代码案例吗？
